js/src/admin/index.js

```diff
@@ -0,0 +1,20 @@
+import { extend } from 'flarum/extend';
+import app from 'flarum/app';
+import PermissionGrid from 'flarum/components/PermissionGrid';
+
+app.initializers.add('mention', () => {
+  extend(PermissionGrid.prototype, 'replyItems', items => {
+    items.add('mention.users', {
+      icon: 'fas fa-at',
+      label: app.translator.trans('flarum-mentions.admin.permissions.mention_users_discussions_label'),
```

Comment:
What are the translations?

Reply:
https://github.com/flarum/lang-english/pull/139, they might need some improvement